### PR TITLE
Breakup

### PR DESCRIPTION
**Pull request by JasonjawsII**
This PR breaks all the diagrams and their previews out of the main scratchpad document so they can more easily be found and maintained on their own.

### CODE REVIEW

File: Boundaries-components.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+### Boundaries, components, and user interactions
+Every data.gov component follows the pattern described below with variance only in the application or data services in use unless otherwise specified.
+
```
Comment (mogul) [inline Boundaries-components.md]: Can you add the preview image here?

File: deployment-change.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+### Deploying a change to production code
+
+The following interactions make a change happen in the production environment. A narrative explanation follows the diagram.
+
```
Comment (mogul) [inline deployment-change.md]: Can you add the preview image here?

File: deployment.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+## Deployment
+Configuration management and deployment for every data.gov component follows the pattern described unless otherwise specified.
+
+Figure 10-1 Deployment Diagram
+
```
Comment (mogul) [inline deployment.md]: Can you add the preview image here?

File: privileged-session.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+### Establishing a privileged session
+The following interactions happen between cloud.gov systems when the data.gov team establishes an authenticated SSH session. See the Cloud Foundry documentation for additional details if necessary (https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-apps.html#other-ssh-access).
+
```
Comment (mogul) [inline privileged-session.md]: Can you add the preview image here?

**Review by mogul — CHANGES_REQUESTED** [review]
The breakout looks good, but let's make sure each file includes the rendered image based on the current PlantUML source code.

File: Boundaries-components.md (hunk 1 above)
Comment (JasonjawsII) [reply Boundaries-components.md]: image

File: deployment.md (hunk 3 above)
Comment (JasonjawsII) [reply deployment.md:5]: image

File: deployment-change.md (hunk 2 above)
Comment (JasonjawsII) [reply deployment-change.md]: image

File: deployment.md (hunk 3 above)
Comment (JasonjawsII) [reply deployment.md]: image

Comment (JasonjawsII) [reply deployment.md]: Need help on this one

File: privileged-session.md (hunk 4 above)
Comment (JasonjawsII) [reply privileged-session.md]: image

**Comment by JasonjawsII** [comment]
Hi Bret,

I added the images but one I need help in downsizing.

Best,

Jason

**Review by mogul — CHANGES_REQUESTED** [review]
When I said "here" I meant "on this line in the Markdown file". You added the images as comments in the review, but not as links in the actual Markdown file... This would mean they wouldn't be visible in the resulting file once the PR is closed, which is where we want them to be for people reviewing/editing in future.

**Review by mogul — APPROVED** [review]

**Pull request merged**